libraries/AP_InertialSensor/AP_InertialSensor.h

```diff
@@ -798,6 +803,29 @@ class AP_InertialSensor : AP_AccelCal_Client
     bool raw_logging_option_set(RAW_LOGGING_OPTION option) const {
         return (raw_logging_options.get() & int32_t(option)) != 0;
     }
+    // if AP_INERTIALSENSOR_FAST_SAMPLE_WINDOW_ENABLED
```

Comment:
Yes, because of the vehicle include protections it can't actually be defined here, the comment is just to indicate that the code is technically controlled